nixos/modules/system/activation/activation-script.nix

```diff
@@ -154,9 +154,15 @@ in
 
     system.activationScripts.tmpfs =
       ''
-        ${pkgs.utillinux}/bin/mount -o "remount,size=${config.boot.devSize}" none /dev
```

Comment:
Yeah, I've tried to do it like this first but got multiple errors after rebuild about nonexistent `.mount` units. 

On August 28, 2016 8:53:44 PM GMT+03:00, Tuomas Tynkkynen <notifications@github.com> wrote:
>> @@ -154,9 +154,15 @@ in
>>  
>>      system.activationScripts.tmpfs =
>>        ''
>> -        ${pkgs.utillinux}/bin/mount -o
>"remount,size=${config.boot.devSize}" none /dev
>
>Meh, apparently not, the code explicitly refuses to create any mount
>units referring to them.
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/NixOS/nixpkgs/pull/17822/files/58d8347ba7d5a517a03fd5209a9e8483dbdea225#r76536291

-- 
Nikolay. 

Sent from a mobile device, please excuse my brevity.